engine/src/main/java/org/terasology/math/MatrixUtils.java

```diff
@@ -241,6 +257,10 @@ public static Matrix4f calcModelViewMatrix(Matrix4f m, Matrix4f vm) {
         return result;
     }
 
+    public static org.joml.Matrix4f calcModelViewMatrix(Matrix4fc m, Matrix4fc vm) {
```

Comment:
Again, doc would be highly appreciated! Which parameters is modified, which is not? I think these are common pitfalls, and conventions that are not very obvious if you're not working with these classes all the time.

Reply:
I don't use the method anyways. it just wraps a multiplication operation where a simple comment would probably be easier to interpret then a method use-case.

Reply:
only one method is calling this and its just a multiplication so I don't really see a value in this method call. I think I'll probably remove it at a later point.